### PR TITLE
J Corp Representative

### PR DESCRIPTION
**Pull request by tile4**
## About The Pull Request
Adds a J corp Representative to the possible representatives you can get. The main gimmicks this rep has is around locking out some of the bad outcomes of RNG, such as increasing the chance to get better loot from gacha, as well as improve chances to get better gear from the well. 

## Why It's Good For The Game
More Representatives is nice to have.

## Changelog
:cl:
add: Added new things
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/trusted_players/representative.dm [hunk 1]
```diff
@@ -64,3 +65,7 @@
 /datum/outfit/job/representative/pcorprepresentative
 	name = "P Corp Representative"
 	uniform = /obj/item/clothing/under/suit/lobotomy/pcorp
+
+datum/outfit/job/representitive/jcorprepresentative
```
Comment (KingSuperBOB) [inline code/modules/jobs/job_types/trusted_players/representative.dm:69]:
```suggestion
/datum/outfit/job/representitive/jcorprepresentative
```
You forgot to add a /

File: ModularTegustation/tegu_items/refinery/crates/_crate.dm [hunk 2]
```diff
@@ -29,11 +29,22 @@
...
+	for(var/upgradecheck in GLOB.jcorp_upgrades)
+		switch(upgradecheck)
+			if("Gacha Chance 4")
+				repmodifier += 3
+			if("Gacha Chance 3")
+				repmodifier += 3
+			if("Gacha Chance 2")
+				repmodifier += 2
+			if("Gacha Chance 1")
+				repmodifier += 2
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/refinery/crates/_crate.dm:47]: are these chances right?
feels a bit weird that level 3 is the same as 4, and 1 the same as 2

**Comment by Kitsunemitsu** [comment]
Wait for Crits; this could be epic. Crit build time.

**Comment by Kitsunemitsu** [comment]
This PR will wait a little longer, as I have some ideas to flesh out J-Corp but it needs more PRs merged first.

**Comment by InsightfulParasite** [comment]
Assigned kirie since she has a plan for this one and it will make sure it doesnt get lost to the annuls of history.

File: ModularTegustation/tegu_items/refinery/crates/_crate.dm [hunk 3]
```diff
@@ -42,6 +53,13 @@
 	if(I.tool_behaviour != TOOL_CROWBAR)
 		return
 
+	rarechance += repmodifier
+	if(veryrarechance)
+		if(istype(src, /obj/structure/lootcrate/workshopallas))
+			veryrarechance += (repmodifier/4)
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/refinery/crates/_crate.dm:59]: I'd prefer is the /4 part is made a variable for each crate.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
While a little barebones, is a great addition to the game. 
One small change to make and the rest looks fine to me.

**Review by Kitsunemitsu — APPROVED** [review]
Looks great to me.
Quite barebones, but what can you do.

**Pull request merged**